src/V3Descope.cpp

```diff
@@ -98,6 +98,8 @@ class DescopeVisitor final : public VNVisitor {
             string::size_type pos;
             if ((pos = name.rfind('.')) != string::npos) name.erase(0, pos + 1);
             return "this->" + name;
+        } else if (scopep->aboveCellp() && scopep->aboveCellp()->isVirtIface()) {
+            return "/*vlSymsp*/" + scopep->aboveCellp()->ifaceVarp()->name();
```

Comment:
You mentioned there are already some cases this doesn't work for, what are they?